minio/provider.go

```diff
@@ -50,7 +50,6 @@ func Provider() *schema.Provider {
 			"minio_ssl": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Default:     false,
 				Description: "Minio SSL enabled (default: false)",
```

Comment:
Should we update also the description here?

```suggestion
				Description: "Minio SSL enabled (default: true)",
```

Reply:
No, the default is still on false, it's just that `Default` overwrote the `DefaultFunc` completely.